### ISSUE TITLE
400 Bad Request error: The value provided for parameter 'query' is invalid

### ISSUE THREAD

**Issue by francovp** [claranet/terraform-datadog-monitors#21]
Hello, I'm getting this errors in version 4.2.x, doesn't happen in version 4.1.2

```
Error: error validating monitor from https://api.datadoghq.com/api/v1/monitor/79946009/validate: 400 Bad Request: {"errors": ["The value provided for parameter 'query' is invalid"]}
│ 
│   with module.k8s-datadog-monitor.module.datadog-monitors-caas-kubernetes-pod.datadog_monitor.pod_phase_status[0],
│   on .terraform/modules/k8s-datadog-monitor.datadog-monitors-caas-kubernetes-pod/caas/kubernetes/pod/monitors-k8s-pod.tf line 1, in resource "datadog_monitor" "pod_phase_status":
│    1: resource "datadog_monitor" "pod_phase_status" {
│ 
╵
╷
│ Error: error validating monitor from https://api.datadoghq.com/api/v1/monitor/79946012/validate: 400 Bad Request: {"errors": ["The value provided for parameter 'query' is invalid"]}
│ 
│   with module.k8s-datadog-monitor.module.datadog-monitors-caas-kubernetes-pod.datadog_monitor.error[0],
│   on .terraform/modules/k8s-datadog-monitor.datadog-monitors-caas-kubernetes-pod/caas/kubernetes/pod/monitors-k8s-pod.tf line 30, in resource "datadog_monitor" "error":
│   30: resource "datadog_monitor" "error" {
│ 
╵
╷
│ Error: error validating monitor from https://api.datadoghq.com/api/v1/monitor/79946008/validate: 400 Bad Request: {"errors": ["The value provided for parameter 'query' is invalid"]}
│ 
│   with module.k8s-datadog-monitor.module.datadog-monitors-caas-kubernetes-pod.datadog_monitor.terminated[0],
│   on .terraform/modules/k8s-datadog-monitor.datadog-monitors-caas-kubernetes-pod/caas/kubernetes/pod/monitors-k8s-pod.tf line 60, in resource "datadog_monitor" "terminated":
│   60: resource "datadog_monitor" "terminated" {
```

My code:

```terraform
module "datadog-monitors-caas-kubernetes-pod" {
  source = "claranet/monitors/datadog//caas/kubernetes/pod"
  version = "4.2.1"

  environment = "prod"
  message       = module.datadog-message-alerting.alerting-message

  prefix_slug = "FIF"
  filter_tags_use_defaults = false
  filter_tags_custom         = "cluster_name:${var.tag_cluster_name}"
}
```

Maybe [11816cf5](https://github.com/claranet/terraform-datadog-monitors/commit/11816cf5f6b449d81a746757a6f993839a0e9e6d) could be the problem?

**Comment by vigohe** [comment]
I also have the same issue! can you please merge it?

**Comment by djagoudel-claranet** [comment]
Fixed with the version 4.2.3
